packages/xrpl/src/models/transactions/metadata.ts

```diff
@@ -29,6 +29,42 @@ interface DeletedNode {
 
 export type Node = CreatedNode | ModifiedNode | DeletedNode
 
+/* eslint-disable @typescript-eslint/no-unnecessary-condition -- Typeguard requires checking if undefined */
```

Comment:
Is there a way to use explicit casting to avoid disabling the rules?

Reply:
HasOwnProperty works for this file - https://stackoverflow.com/a/73810861